### DISCUSSION TITLE
只学了Java, 还没学Maven, SpringBoot, 能搞一个bot出来吗

### DISCUSSION THREAD

**Discussion by learningAHAIC** [simple-robot/Discussions#86] [酒馆]
### 涉及的编程语言

Java

### 涉及的组件库

mirai (QQ)

### 疑问描述

大佬们，我是个刚开始学编程2年的学生，现在只会写写java刷刷力扣，刚寻思写点什么能用上的，想挑战一下qq机器人，结果发现我学的太少了，什么maven，什么springBoot，我啥都没学过，就我现在的水平能用这个架构写出一个qq机器人吗？在b站上看了simbot的教程，结果不知道是不是用vscode配置环境的时候出了什么问题，pom.xml里的内容比视频里少好多，能出一个用vscode从配置环境开始的教程吗？感谢大佬了！

**Comment by yujumpjump** [comment]
你可以试试使用原生的mirai来进行开发，maven只是用来更好的组织你开发使依赖的jar，而且既然你使用java开发为什么不使用idea工具呢

**Comment by ForteScarlet** [comment]
很久不用vscode了我忘了怎么用了，而且天天找工作没什么心情录视频
maven之类的东西并不难，随便找个“用maven搭建项目”的文章帖子看看会用就行

> 就我现在的水平能用这个架构写出一个qq机器人吗

虽然simbot并不强制要求用springboot，但是用spring对新手更友好（个人见解）
springboot基础的使用很简单，找几个文章视频的照猫画虎就行了，也可以暂时不去管什么原理，大不了不懂的问问老师啥的
